### PR TITLE
Adds new emotes, allows for CAPITAL LETTERS to work in emotes.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
so *frown *Frown and *FROWN all work instead of just *frown. 

Thanks to shaps for the actual code for making that work... I can't code.

I don't think this is worth change logging

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -94,6 +101,10 @@
 			message = "<B>[src]</B> frowns."
 			m_type = 1
 
+		if ("gag","gags")
+			message = "<B>[src]</B> gags!"
+			m_type = 2
```
Comment (KazeEspada) [inline code/modules/mob/living/emote.dm]: GAGS ON DIS DICK

**Comment by xxalpha** [comment]
>adding more stuff on the emote pile

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -222,10 +264,29 @@
...
 
+		if ("stretch","stretches")
+			message = "<B>[src]</B> stretches their arms."
+			m_type = 2
+
 		if ("sulk","sulks")
 			message = "<B>[src]</B> sulks down sadly."
 			m_type = 1
 
+		if ("surrender","surrender")
```
Comment (MrStonedOne) [inline code/modules/mob/living/emote.dm]: you forgot the s

**Comment by Shapsy** [comment]
*Grin

**Comment by MMMiracles** [comment]
*me uses the 'me'  command to replicate everything already here

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -222,10 +264,29 @@
...
+			if(sleeping)
+				return //Can't surrender while asleep.
+			Weaken(20) //So you can't resist.
+			m_type = 1
+
+		if ("faint","faints")
+			message = "<B>[src]</B> faints."
+			if(sleeping)
+				return //Can't faint while asleep
+			SetSleeping(10) //Short-short nap
```
Comment (Iamgoofball) [inline code/modules/mob/living/emote.dm]: use Paralyse()

**Comment by Iamgoofball** [comment]
fair warning I'm adding datum emotes soon(tm)

File: code/modules/mob/living/emote.dm (hunk 3 above)
Comment (TechnoAlchemisto) [inline code/modules/mob/living/emote.dm]: what's the difference

Comment (Iamgoofball) [inline code/modules/mob/living/emote.dm]: standards

**Comment by Supermichael777** [comment]
*me can't be forced for miles to do the silent version

**Comment by Cheridan** [comment]
>so *frown *Frown and *FROWN all work instead of just *frown. 

What about *fRoWn?

**Comment by TechnoAlchemisto** [comment]
That works too!

**Comment by Incomptinence** [comment]
Well I do like emotes

**Pull request merged**